### PR TITLE
Add a tab for MD Anderson heatmaps (where available)

### PR DESCRIPTION
**Pull request by alisman**
Call service to determine availability of heatmap for given patient id.  Where available, show iframe with heatmap

The following URL will allow you to view:
http://localhost:3000/#/patient?studyId=ucec_tcga_pub&caseId=TCGA-A5-A0G2&tab=tissueImageTab&navCaseIds=TCGA-A5-A0G1%2CTCGA-A5-A0G2%2CTCGA-A5-A0G3%2CTCGA-A5-A0G5

Also, fixed hardcoded patient id in tissue images

### CODE REVIEW

File: src/pages/patientView/clinicalInformation/PatientViewPageStore.ts [hunk 1]
```diff
@@ -245,6 +245,27 @@ export class PatientViewPageStore
...
+        invoke: async() => {
+
+            let resp: any = await request.get(`//bioinformatics.mdanderson.org/dyce?app=chmdb&command=participant2maps&participant=${this.patientId}`);
+
+            const parsedResp: any = JSON.parse(resp.text);
+
+            // filecontent array is serialized :(
+            const fileContent: string[] = JSON.parse(parsedResp.fileContent);
+
+            console.log("moo",fileContent.length);
```
Comment (adamabeshouse) [inline src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: lol

File: src/pages/patientView/clinicalInformation/PatientViewPageStore.ts [hunk 2]
```diff
@@ -245,6 +245,27 @@ export class PatientViewPageStore
 
     }, []);
 
+    readonly MDAndersonHeatMap = remoteData({
```
Comment (adamabeshouse) [inline src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: This is sort of nitpicky but, it seems like you're not actually using this fetched data, you just want to know it exists. So to keep things simple and direct, I'd prefer to see this be called `MDAndersonHeatMapAvailable` and have it return the boolean `fileContent.length > 0`, default `false`

File: src/pages/patientView/clinicalInformation/PatientViewPageStore.ts (hunk 1 above)
Comment (alisman) [inline src/pages/patientView/clinicalInformation/PatientViewPageStore.ts]: sigh

File: src/pages/patientView/PatientViewPage.tsx [hunk 3]
```diff
@@ -543,10 +543,18 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </MSKTab>)
                     }
 
+
+                    {  (patientViewPageStore.MDAndersonHeatMapAvailable.isComplete && patientViewPageStore.MDAndersonHeatMapAvailable.result ) &&
```
Comment (adamabeshouse) [inline src/pages/patientView/PatientViewPage.tsx]: couldn't this just be `patientViewPageStore.MDAndersonHeatMapAvailable.result` now?

**Pull request merged**